command/src/logging/logs.rs

```diff
@@ -293,6 +308,14 @@ impl InnerLogger {
 
         if let Err(e) = io_result {
             println!("Could not write access log to {}: {e:?}", backend.as_ref());
+            println!(
+                "trying to revive the backend of access logs to {:?}, or defaulting to {}",
+                self.access_logs_target, self.log_target
+            );
+            backend.revive(self.access_logs_target.as_ref().unwrap_or(&self.log_target));
+            false
+        } else {
+            true
```

Comment:
this `log_access` function returns a bool because it is called by the `_log_access` macro, that handles success or failure on its own, with custom actions defined where the macros are called. Typically:

```rust
        info_access!(
            on_failure: { incr!("unsent-access-logs") },
            // ...
        );
```

the purpose of the `log_access` function is:
- try to send an access log to the appropriate backend
- try to revive the backend if needed
- handle the errors by logging them with print statements
- return a bool to notify the macro